src/handlebars/installers.handlebars

```diff
@@ -0,0 +1,169 @@
+          deployment. Follow these steps:
+
+        <p>1. Download the <tt>.msi</tt> file.</p>
+        <p>2. Choose the features that you want to install, which are shown in the following table:
+            <br>
+              <table>
+
+                <tr>
+                  <th>Feature</th>
+                  <th>Description</th>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureMain</tt></td>
+                  <td>Core OpenJDK installation (DEFAULT)</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureEnvironment</tt></td>
+                  <td>Update the <strong>PATH</strong> environment variable (DEFAULT)</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureJarFileRunWith</tt></td>
+                  <td>Associate <tt>.jar</tt> files with Java applications (DEFAULT)</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureJavaHome</tt></td>
+                  <td>Update the <strong>JAVA_HOME</strong> environment variable</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureIcedTeaWeb</tt></td>
+                  <td>Install <a href="https://www.github.com/adoptopenjdk/icedtea-web" target="_blank">IcedTea-Web</a></td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureJNLPFileRunWith</tt></td>
+                  <td>Associate <tt>.jnlp</tt> files with IcedTea-web</td>
+                </tr>
+
+                <tr>
+                  <td><tt>FeatureOracleJavaSoft</tt></td>
+                  <td>Updates registry keys HKLM\SOFTWARE\JavaSoft\</td>
+                </tr>
+
+              </table>
+        </p>
+
+        <p>Note: <tt>FeatureOracleJavaSoft</tt> can be used to prevent Oracle Java launching from PATH when AdoptOpenJDK is uninstalled. Reinstall Oracle Java if you need
+        to restore the Oracle registry keys.</p>
+
+        <p>Optional parameters can be used that group some of the features together:
+            <br>
+            <table>
+              <tr>
+                <th>Parameter</th>
+                <th>Features</th>
+              </tr>
+
+              <tr>
+                <td>INSTALLLEVEL=1</td>
+                <td>FeatureMain,FeatureEnvironment,FeatureJarFileRunWith</td>
+              </tr>
+
+              <tr>
+                <td>INSTALLLEVEL=2</td>
+                <td>FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome</td>
+              </tr>
+            </table>
+        </p>
+
+        <p>3. Run the command on the target workstation.</p>
+
+        <p>The following example silently installs OpenJDK, updates the PATH, associates <tt.jar</tt> files with Java applications and defines JAVA_HOME:</p>
+        <p><tt>msiexec /i &lt;package&gt;.msi ADDLOCAL=FeatureJavaHome /quiet</tt></p>
```

Comment:
To be coherent with previous comment :

> The following example silently installs OpenJDK, updates the PATH, associates <tt.jar</tt> files with Java applications and defines JAVA_HOME:

I must be with all feature in the list 

><p><tt>msiexec /i &lt;package&gt;.msi ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome /quiet</tt></p>